packages/editor/src/components/editors/menu/HeaderMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import './HeaderMenu.css';
+import { Root, Trigger, Portal, Content, Item, Arrow } from '@radix-ui/react-dropdown-menu';
+import IvyIcon from '../../widgets/IvyIcon';
+import { IvyIcons } from '@axonivy/editor-icons';
+import { useDataContext, useEditorContext, useTheme } from '../../../context';
+import { deepEqual } from '../../../utils/equals';
+import { useEffect, useState } from 'react';
+import { Button } from '../../../components/widgets';
+
+const DropdownMenu = () => {
+  const { data, initData, setData } = useDataContext();
+  const { mode, setMode } = useTheme();
+  const { editorRef } = useEditorContext();
+  const [editor, setEditor] = useState<HTMLElement | null>(null);
+
+  useEffect(() => {
+    setEditor(editorRef.current);
+  }, [editorRef]);
+
+  return (
+    <Root modal={false}>
+      <Trigger asChild>
+        <Button icon={IvyIcons.AllElements} aria-label='Menu' />
+      </Trigger>
+
+      <Portal container={editor}>
```

Comment:
Portal? ok

Reply:
Yes, this defines where the menu is rendered, now it is attached to the editor-root div. This makes sure that it is not cut from everything else.